codecov.yml

```diff
@@ -0,0 +1 @@
+
```

Comment:
What's the purpose of this file? 🤔

Reply:
One great feature that Codecov offers is the ability to add coverage reports right in your github pull requests. All you need to do is add a codecov.yml file in the root of your project. The file can be empty, it just needs to exist.